cmd/cli/serve/serve.go

```diff
@@ -286,13 +281,31 @@ func serve(cmd *cobra.Command) error {
 		return fmt.Errorf("error starting node: %w", err)
 	}
 
+	startWebUI, err := config.Get[bool](types.NodeWebUIEnabled)
+	if err != nil {
+		return err
+	}
+
 	// Start up Dashboard
 	if startWebUI {
+		listenPort, err := config.Get[int](types.NodeWebUIPort)
+		if err != nil {
+			return err
+		}
+
+		// If we get the default value for an int, use 80 instead.
+		if listenPort == 0 {
+			listenPort = 80
+		}
```

Comment:
we can remove this now. The default is used from the config via 138c8e9dbc61aac9126f5ae4e980fa2e045b6b31

Reply:
Ah. We could add a linter step to make sure that all generate commands are up to date (i.e. don't produce a diff)

Reply:
https://github.com/bacalhau-project/bacalhau/pull/3133#pullrequestreview-1789760388
😄😄😄

Reply:
fwiw That wouldn't have completely caught the problem here, the other part that needed to change was updating our default config structures and using the defaults for the value default value. But would be good to have regardless.